src/style/adwaitastyle.cpp

```diff
@@ -2778,10 +2778,12 @@ QSize Style::spinBoxSizeFromContents(const QStyleOption *option, const QSize &co
     size.rwidth() += Metrics::Button_ItemSpacing;
 
     // FIXME this shouldn't be needed but apparently some width is still missing
-    size.rwidth() += size.height() / 2;
+    // reduce to 1 / 4 sems enough, was 1 / 2
+    size.rwidth() += size.height() / 4;
 
     // set minimum size
-    size.setHeight(qMax(size.height(), int(Metrics::SpinBox_MinHeight)));
+    // size.height() appears way too much - so removed
+    size.setHeight(int(Metrics::SpinBox_MinHeight));
```

Comment:
Then it means it needs to be fixed above if the original height is too much.

Reply:
No comprendo :)